### ISSUE TITLE
Can't use chest cart inventory

### ISSUE THREAD

**Issue by Gwolfski** [bergerkiller/TrainCarts#32]
When I right click on a chest cart, the GUI doesn't open. Tried in creative and survival. I am /op on the server. The train has default properties and is just a hand-placed chest cart. In spectator mode, I can view the inventory but not change it, as expected.

**Comment by bergerkiller** [comment]
Yeah this was fixed in dev https://ci.mg-dev.eu/job/TrainCarts/283/

Will be part of next release too. Dont use the latest build as its a bit bleeding edge, but the one I linked is stable.

**Comment by Gwolfski** [comment]
ah, thanks!
